src/application-insights/azext_applicationinsights/_params.py

```diff
@@ -84,3 +84,36 @@ def load_arguments(self, _):
     with self.argument_context('monitor app-insights component linked-storage') as c:
         c.argument('storage_account_id', options_list=['--storage-account', '-s'], validator=validate_storage_account_name_or_id,
                    help='Name or ID of a linked storage account.')
+
+    with self.argument_context('monitor app-insights component continues-export list') as c:
+        c.argument('application', id_part=None)
+
+    with self.argument_context('monitor app-insights component continues-export') as c:
+        c.argument('record_types', nargs='+',
+                   arg_type=get_enum_type(
+                       ['Requests', 'Event', 'Exceptions', 'Metrics', 'PageViews', 'PageViewPerformance', 'Rdd',
+                        'PerformanceCounters', 'Availability', 'Messages']),
+                   help='The document types to be exported, as comma separated values. Allowed values include \'Requests\', \'Event\', \'Exceptions\', \'Metrics\', \'PageViews\', \'PageViewPerformance\', \'Rdd\', \'PerformanceCounters\', \'Availability\', \'Messages\'.')
+        c.argument('dest_sub_id', arg_group='Destination',
+                   help='The subscription ID of the destination storage account.')
+        c.argument('dest_account', validator=validate_dest_account, arg_group='Destination',
+                   help='The name of destination storage account.')
+        c.argument('dest_container', arg_group='Destination', help='The name of the destination storage container.')
+        c.argument('dest_sas', arg_group='Destination',
+                   help='The SAS token for the destination storage container. It must grant write permission.')
+        c.argument('dest_type', arg_group='Destination', arg_type=get_enum_type(['Blob']),
+                   help='The Continuous Export destination type. This has to be \'Blob\'.')
+        c.argument('is_enabled', arg_type=get_three_state_flag(return_label=True),
```

Comment:
>get_three_state_flag [](start = 42, length = 20)

Use 'for' clause to avoid duplication, for example,

    for scope in ['update', 'show', 'delete']:
        with self.argument_context('monitor app-insights component continues-export {}'.format(item)) as c:
            c.argument('export_id', options_list=['--id'],
                   help='The Continuous Export configuration ID. This is unique within a Application Insights component.')